modules/programs/md2pdf/default.yaml

```
@@ -0,0 +1,49 @@
+title-meta: "${title}"
+author-meta: "${author}"
+lang: en-US
+
+# PDF generation settings
+pdf-engine: xelatex
+template: eisvogel
+
+# Typography matching Obsidian
+variables:
+  mainfont: "Inter"
+  sansfont: "Inter"  
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)